assessment_software/src/app/navbar/navbar.component.css

```diff
@@ -0,0 +1,15 @@
+.navbar-container{
+    @apply  bg-white border-gray-200 border border-solid rounded px-2 text-sm font-sans fixed top-1 left-1 right-1 grid grid-cols-4 gap-2 items-center;
+}
```

Comment:
the navbar is to tight with the contents inside it. please add some padding

![image](https://github.com/Aben-Bel/assessment_software/assets/44743368/d0b9648f-0da0-4fa8-9573-5fe73edc2fde)